types/stringify-object/index.d.ts

```diff
@@ -2,6 +2,7 @@
 // Project: https://github.com/yeoman/stringify-object
 // Definitions by: Piotr Błażejewicz <https://github.com/peterblazejewicz>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// Minimum TypeScript Version: 4.5
```

Comment:
```suggestion
```
please remove this, won't be necessary if you rewrite imports in test file

Reply:
This is what `npm test stringify-object` told me to do.